### ISSUE TITLE
UX

### ISSUE THREAD

**Issue by Request-Off-Work** [terryum/awesome-deep-learning-papers#29]
Can you please add more "information scent" to the links? A one sentence summary of the paper would be very helpful. Should make the repo a very interesting read :) Thanks!

**Comment by terryum** [comment]
Yeah. I think it's a very good idea. I cannot do that all by myself, I may need others' contributions as well. Also I hope to have a UX which do not need to move to another paper to see just one-line summary. (And you know, it should not make the list look dirty. Any ideas...?

**Comment by Request-Off-Work** [comment]
I think it's brilliant that you currated the list by number of citations. I can try to contribute but I'm not an expert at machine learning yet... And I agree that it won't look as polished if it's a constant WIP. As a designer, I need to present two or more ideas for the dialog to go smoothly. Here's what I'm thinking.

Leave it as it is.

Create a git branch for people to contribute to

Create a seperate md for people to edit.

You could make the links lead to <a href="#description"> and make a table of contents sort of thing.

Those are just some ideas! Let me know how you feel. And yes, I'd help summarize a few articles if I have some free time. Good luck and "awesome" job on your awesome list :)
